cayleypy/permutation_utils.py

```diff
@@ -21,3 +21,24 @@ def inverse_permutation(p: Sequence[int]) -> list[int]:
 
 def is_permutation(p):
     return sorted(list(p)) == list(range(len(p)))
+
+
+def transpositions(n):
```

Comment:
Call this all_transpositions and all type annotation.

Also please change "all_transpositions" case in graphs_lib.prepare_graph to use this.

I would normally also ask to add a test, but if you use it in prepare_graphs, it becomes covered by existing tests.